app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,47 @@
+module Admin
+  class UsersController < ApplicationController
+    def show
```

Comment:
would be nice to add rubocop and run it against whole codebase cause it looks messy right now

Reply:
It's added. @LukBro as a reminder, you run the linter from the console level using `rubocop -a` it should fix stuff like missing spaces between method definitions etc.